package.json

```diff
@@ -18,6 +18,7 @@
   "devDependencies": {
     "@patternfly/react-core": "^4.202.16",
     "@patternfly/react-table": "^4.71.16",
+    "@patternfly/react-virtualized-extension": "^4.53.2",
```

Comment:
@vojtechszocs are we expecting to have to add new dependencies in both the root package.json and the sub-package.json? That seems like a lot of overhead....